pkg/cosign/tuf/client.go

```diff
@@ -46,6 +47,13 @@ type TUF struct {
 	client  *client.Client
 	targets targetImpl
 	local   client.LocalStore
+	remote  client.RemoteStore
+}
+
+// RemoteCache contains information to cache on the location of the remote
+// repository.
+type remoteCache struct {
+	Mirror string `json:"mirror"`
```

Comment:
To avoid exporting, should we use lowercase `mirror`? Since `remoteCache` isn't exported, we shouldn't need to export the struct variable

Reply:
If it's not exported json won't serialize it :/

Reply:
Ah that's unfortunate, oh well